### PR TITLE
ci: update labeling strategy

### PR DESCRIPTION
**Pull request by jorgepiloto**
This pull-request adds the `fix` label to the list of labels. The reason is that, right now, we are labeling with `bug` pull-requests that are actually fixing stuff.

### CODE REVIEW

**Review by Revathyvenugopal162 — APPROVED** [review]

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
Thanks for the changes ! Can you extend them to update the labeler to match your changes ?
See https://github.com/ansys/ansys-sphinx-theme/blob/29733af619815aece9b00838214fa630f0ae6abe/.github/workflows/label.yml#L63

**Comment by jorgepiloto** [comment]
> Thanks for the changes ! Can you extend them to update the labeler to match your changes ? See
> 
> https://github.com/ansys/ansys-sphinx-theme/blob/29733af619815aece9b00838214fa630f0ae6abe/.github/workflows/label.yml#L63

Good point, @SMoraisAnsys. I am taking advantage of this to update the whole labeling strategy.

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -1,100 +0,0 @@
...
-    permissions:
-      contents: write
-      pull-requests: write
-    runs-on: ubuntu-latest
-    steps:
-    - uses: ansys/actions/doc-changelog@v8
-      with:
-        token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
-        bot-user: ${{ secrets.PYANSYS_CI_BOT_USERNAME }}
-        bot-email: ${{ secrets.PYANSYS_CI_BOT_EMAIL }}
```
Comment (RobPasMue) [inline .github/workflows/label.yml:100]: You just got rid of the changelog fragment generation. Please revert this @jorgepiloto

Comment (RobPasMue) [reply .github/workflows/label.yml:100]: Maybe we can add it to the main workflow now, and make it dependent on the labeler like you did for code-style and doc-style

Comment (jorgepiloto) [reply .github/workflows/label.yml:100]: Ah! Thanks for pointing this out. Let me add it to the main workflow file.

**Review by RobPasMue — APPROVED** [review]
LGTM!

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: .github/labels.yml [hunk 2]
```diff
@@ -1,38 +1,91 @@
-- name: bug
-  description: Something isn't working
+# The labels declared in this file are the ones avialables
```
Comment (SMoraisAnsys) [inline .github/labels.yml:1]:
```suggestion
# The labels declared in this file are the ones availables
```

File: .github/labels.yml [hunk 3]
```diff
@@ -1,38 +1,91 @@
...
+
+- name: 'style:code'
+  description: Improvements related to code style
+  color: dcbeff
+
+- name: 'style:docs'
+  description: Improvements related to doc style
+  color: dcbeff
+
+- name: 'style:branch-name'
```
Comment (SMoraisAnsys) [inline .github/labels.yml:28]: Can we add a branch name rule instead of having such label style ?

File: .github/labels.yml [hunk 4]
```diff
@@ -1,38 +1,91 @@
...
 
-- name: hacktoberfest-accepted
-  description: Hacktoberfest accepted contribution
-  color: 62ca50
+- name: 'dependencies:pyproject'
+  description: Related with project file dependencies
+  color: fabed4
+
+- name: 'dependencies:ci'
+  description: Related with pipelines dependencies
```
Comment (SMoraisAnsys) [inline .github/labels.yml:75]:
```suggestion
  description: Related to project file dependencies
  color: fabed4

- name: 'dependencies:ci'
  description: Related to pipelines dependencies
```
For consistency with other descriptions

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, I left minor changes. One might lead to some code removal.

File: .github/labels.yml (hunk 3 above)
Comment (jorgepiloto) [reply .github/labels.yml:28]: Done.

**Pull request merged**